GDJS/Runtime/runtimeobject.ts

```diff
@@ -70,6 +70,10 @@ namespace gdjs {
     //Variables:
     protected _variables: gdjs.VariablesContainer;
 
+    //Effects:
+    protected _effects: EffectData[];
+    protected _filters: Record<string, PixiFiltersTools.Filter> = {}; 
```

Comment:
Note: we have here "PixiFiltersTools.Filter", which is PixiJS specific, but there is nothing inside the runtime object actually PixiJS specific, as everything is going through the object effects manager. So this is good! And we can always change the typing later without breaking anything.

Reply:
We might just want to rename this to `_rendererEffects`. Because "filters" is a PixiJS specific term, while here it's a table that contains the effects of the renderer (i.e: the effects of PIXI, i.e: the PIXI filters).
Other renderers in the future might have a different name ("shaders", etc.., so keeping the GDevelop term ("effect") is in theory better)